cmd/rollkit/commands/intercept.go

```diff
@@ -125,3 +165,14 @@ func RunRollupEntrypoint(rollkitConfig *rollconf.TomlConfig, args []string) erro
 
 	return nil
 }
+
+func parseFlag(args []string, flag string) string {
```

Comment:
aren't we having primitives for flag parsing?

Reply:
this doesn't take into account --flagname=value

Reply:
@yarikbratashchuk REF: https://github.com/rollkit/rollkit/pull/1911#discussion_r1835021377

let me know if this addresses your issue. We can handle other questions offline as follow ups.